flowcraft/generator/templates/seqsero2_assembly.nf

```diff
@@ -0,0 +1,54 @@
+process seqsero2_assembly_{{ pid }} {
+    // Send POST request to platform
+    {% include "post.txt" ignore missing %}
+
+    tag { sample_id }
+    errorStrategy { task.exitStatus == 120 ? 'ignore' : 'ignore' }
+    publishDir path: "results/typing/seqsero2_assembly/${sample_id}/", mode: 'symlink', overwrite: true, pattern: 'Seq*.txt'
```

Comment:
You don't need to use mode: 'symlink' since it is the default.

```suggestion
    publishDir path: "results/typing/seqsero2_assembly/${sample_id}/", overwrite: true, pattern: 'Seq*.txt'
```

Also note that with `overwrite: true` you are saying that each time the script is executed (using resume) the directory is overwrite, right? Is that what you intend?

Reply:
The `mode: 'symlink'` was there because I'm a little bit control freak! But as you pointed out, it's not required. I choose to use `overwrite: true` to be sure that the results present there are really from the last run, but I'm not sure if this is needed.

Reply:
It shouldn't be necessary but it's fine.

Reply:
I'll change it